src/common/pico_binary_info/CMakeLists.txt

```diff
@@ -10,14 +10,18 @@ endif()
 target_link_libraries(pico_binary_info INTERFACE pico_binary_info_headers)
 
 # pico_set_program_name(TARGET name)
-# Set the program name for the target
+# \brief\ Set the program name for the target
+#
+# \param\ name The program name to set
```

Comment:
Why is there a `\param\ name` but not a `\param\ TARGET` ?

Reply:
I removed all the TARGET params as it means the same everywhere so I don't think it needs to be documented for every function, and loads of CMake functions have TARGET (eg `add_executable` which is needed for every executable) so it should be clear enough